app/assets/stylesheets/layouts/_col_marginless.scss

```diff
@@ -0,0 +1,42 @@
+.l-2col-even--marginless {
```

Comment:
This is a bit of a hacky name. Basically what we wanted was a 2col layout that didn't add 'margin' to the left side of the first and right side of the last.